lib/pkcs11/pkcs11_token.c

```diff
@@ -428,8 +428,8 @@ CK_RV pkcs11_token_get_info(CK_SLOT_ID slotID, CK_TOKEN_INFO_PTR pInfo)
     pInfo->ulMinPinLen = 0;
     pInfo->flags = CKF_RNG;// | CKF_LOGIN_REQUIRED;
 
-    pInfo->ulMaxSessionCount = 1;
-    pInfo->ulMaxRwSessionCount = 1;
+    pInfo->ulMaxSessionCount = PKCS11_MAX_SESSIONS_ALLOWED;
```

Comment:
Based on bug reports, there is risk that setting this to PKCS11_MAX_SESSIONS_ALLOWED might break something?